deploy/scripts/pre-upgrade-dry-run.sh

```diff
@@ -82,8 +82,8 @@ wait_for_database() {
     podman_args+=("--timeout=${DB_WAIT_TIMEOUT}" "--sleep=${DB_WAIT_SLEEP}")
 
     if ! "${PODMAN}" run "${podman_args[@]}"; then
-        echo "[flightctl] database wait failed; skipping dry-run"
-        exit 0
+        echo "[flightctl] database wait failed; aborting pre-upgrade dry-run and stopping upgrade"
```

Comment:
database wait is implemention details can we also clarify to the user that the database is not ready